test/box/gh-5645-several-iproto-threads.lua

```diff
@@ -2,10 +2,15 @@
 
 require('console').listen(os.getenv('ADMIN'))
 
+local iproto_threads = arg[1]
+if iproto_threads == "negative" then
+	iproto_threads = -1
```

Comment:
Just wonder, why can't we simply pass -1 via test_run:cmd?

Reply:
I try to do it, and i have some strange error ` builtin/internal.argparse.lua:120: bad argument #3: ID not valid`
I try to do it for another tests, to check that it is not a problem of `iproto_threads` option and have the same result.